mailchimp_subscriber.py

```diff
@@ -70,7 +83,10 @@ def load_users(users_file):
             # Note that this isn't testing for multiple appearances of the same
             # client. If theres more than one, it takes the last appearence.
             try:
-                clients[row[EMAIL_COL]] = Client(row)
+                if (len(row) >= 3):
+                    clients[row[EMAIL_COL]] = Client(row[EMAIL_COL],
```

Comment:
nit: this is a bit confusing to read. Suggestion on a clearer, but more verbose approach:
```
client = Client(...)
clients[client.email_address] = client